linera-views/src/scylla_db.rs

```diff
@@ -328,41 +335,138 @@ impl ScyllaDbClientInternal {
         client.1.clone()
     }
 
-    async fn new(
-        restart_database: bool,
+    /// Tests if a table is present in the database or not
+    pub async fn test_existence_table(
```

Comment:
In English that would be: `test_table_existence` or `test_if_table_exists`

Reply:
Ok, I used `test_table_existence`.